tests/chart_tests/test_astronomer_commander.py

```diff
@@ -13,8 +22,23 @@ class TestAstronomerCommander:
     def test_astronomer_commander_deployment_default(self, kube_version):
         """Test that helm renders a good deployment template for
         astronomer/commander."""
+        values = {
+            "astronomer": {
+                "airflowChartVersion": "11.5.10",
```

Comment:
We should test custom values with a value that is obviously wrong.

```suggestion
                "airflowChartVersion": "99.88.77",
```